src/components/cylc/Toolbar.vue

```diff
@@ -110,11 +143,43 @@ export default {
         this.responsiveInput = true
       }
     },
-    onClickPause () {
-      // TODO: implement the pause action
+    onClickReleaseHold () {
+      const vm = this
+      if (this.isHeld) {
+        // release
+        this.$apollo.mutate({
+          mutation: RELEASE_WORKFLOW,
+          variables: {
+            workflow: this.workflow.id
+          }
+        }).then(() => {
+          vm.isStopped = false
+          vm.isHeld = false
+        })
+      } else {
+        // hold
+        this.$apollo.mutate({
+          mutation: HOLD_WORKFLOW,
+          variables: {
+            workflow: this.workflow.id
+          }
+        }).then(() => {
+          vm.isStopped = false
```

Comment:
Sure! Good idea :+1:

Reply:
@oliver-sanders really good idea. Now when I hold a workflow in the command line, the UI changes its state too on the next poll.

However, we don't have the stopped state yet (see #221). So I've added a `FIXME` marker so we revisit it once we have decided how to handle the stopped workflows.

Note too, that upon refreshing the page after stopping it, you should get a blank page. Even if we moved that information to the central store, it would still give us a blank page, as the central store relies on the backend for the data.

Does that sound reasonable? i.e. move the isStopped state out of the component once we have fixed @221?

Reply:
Brilliant!

Stopped suites will need to be handled by the UI Server, that's definitely work for later...

Reply:
Thanks! Just need to add some tests to promote it from draft to PR then :+1: will start working on it tomorrow.